src/ShopBundle/Resources/translations/messages.de.xliff

```diff
@@ -159,12 +159,12 @@
 
             <trans-unit id="chameleon_system_shop.payment_paypal_express.error_guest_user_with_id">
                 <source>chameleon_system_shop.payment_paypal_express.error_guest_user_with_id</source>
-                <target>Benutzerdaten nicht ok. Ein User MIT ID ist als nicht Angemeldet markiert</target>
+                <target>Benutzerdaten nicht OK. Ein User MIT ID ist als nicht angemeldet markiert</target>
             </trans-unit>
 
             <trans-unit id="chameleon_system_shop.payment_easy_cash.error_unable_to_confirm_payment">
                 <source>chameleon_system_shop.payment_easy_cash.error_unable_to_confirm_payment</source>
-                <target>Konnte die Zahlungsdetails beim PayMaster auch nach 3 Versuchen nicht bestätigen. Bitte versuchen Sie es später erneut</target>
+                <target>Konnte die Zahlungsdetails beim PayMaster auch nach 3 Versuchen nicht bestätigen. Bitte versuchen Sie es später erneut.</target>
```

Comment:
Who?

Reply:
It's the name of the payment provider - not perfectly expressed for end-users, but OK I think.